src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1330,14 +1359,28 @@ class ContentExplorer extends Component<Props, State> {
     };
 
     /**
-     * Handle pagination changes
+     * Handle pagination changes for offset based pagination
      *
      * @param {number} newOffset - the new page offset value
      */
     paginate = (newOffset: number) => {
         this.setState({ currentOffset: newOffset }, this.refreshCollection);
     };
 
+    /**
+     * Handle pagination changes for marker based pagination
```

Comment:
dont bother with docs anymore, we have stopped adding them unless some explanation is needed